### PR TITLE
Update app

### PR DESCRIPTION
**Pull request by uturuncoglu**
This updates the UFS app to solve the remaining issue of CHGRES failure. The PR is tested with the C786 and the whole workflow runs without any problem on Cheyenne. So, it is ready to test broadly.

### CODE REVIEW

File: Externals.cfg [hunk 1]
```diff
@@ -2,26 +2,34 @@
 schema_version = 1.0.0
 
 [model]
-#branch = release/public-v1
-tag = ufs-v1.0.0
+# branch = release/public-v1
```
Comment (climbfuji) [inline Externals.cfg:5]: I understand that anything in `Externals.cfg` that is not using a fixed tag will be updated after we conducted all our testing (and we can compare the code manually using `meld` or something like that to make sure that after checking out it is identical).

**Comment by uturuncoglu** [comment]
@ligiabernardet I created PR but it also includes changes in the documentation. If there is something wrong, just let me know.

**Comment by ligiabernardet** [comment]
I am confused. I thought what we needed was a PR to release/public-v1 (not master).

**Comment by uturuncoglu** [comment]
@ligiabernardet do you want to do it before testing more broadly? If so, I could change it.

**Comment by uturuncoglu** [comment]
@ligiabernardet sorry, you are right. I am fixing it

**Comment by uturuncoglu** [comment]
@ligiabernardet fixed. sorry for confusion.

**Review by ligiabernardet — APPROVED** [review]

File: Externals.cfg (hunk 1 above)
Comment (uturuncoglu) [reply Externals.cfg:5]: @climbfuji After testing this version of app, I'll create tags for CIME and FV3 interface and update the Externals.cfg to make the final PR. We have already using release tags for model and NEMS interface.

**Pull request merged**